install/config/nginx.dist

```diff
@@ -52,6 +52,10 @@ server {
 		return 403;
 	}
 
+	location /cache {
```

Comment:
This causes nginx to not pass through missing files to index.php! So this works up until your simplecache time is changed, at which point you get 404s from all cache URLs :-1: